src/Client/Grpc/GrpcDurableEntityClient.cs

```diff
@@ -0,0 +1,146 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Google.Protobuf.WellKnownTypes;
+using Microsoft.DurableTask.Client.Entities;
+using Microsoft.DurableTask.Entities;
+using Microsoft.Extensions.Logging;
+using static Microsoft.DurableTask.Protobuf.TaskHubSidecarService;
+using DTCore = DurableTask.Core;
+using P = Microsoft.DurableTask.Protobuf;
+
+namespace Microsoft.DurableTask.Client.Grpc;
+
+/// <summary>
+/// The client for entities.
+/// </summary>
+class GrpcDurableEntityClient : DurableEntityClient
+{
+    readonly TaskHubSidecarServiceClient sidecarClient;
+    readonly GrpcDurableTaskClient durableTaskClient;
+    readonly ILogger logger;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="GrpcDurableEntityClient"/> class.
+    /// </summary>
+    /// <param name="durableTaskClient">The durable Task client.</param>
+    /// <param name="sidecarClient">The client for the GRPC connection to the sidecar.</param>
+    /// <param name="logger">The logger for logging client requests.</param>
+    public GrpcDurableEntityClient(GrpcDurableTaskClient durableTaskClient, TaskHubSidecarServiceClient sidecarClient, ILogger logger)
```

Comment:
The only thing used from `GrpcDurableTaskClient` is the `DataConverter` - should we just take that in directly?

Reply:
Sure, why not.